packages/measure/src/types.ts

```diff
@@ -20,6 +20,9 @@ export interface MeasureResults {
   /** Array of measured render/execution durations for each warmup run   */
   warmupDurations: number[];
 
+  /** Array of statistical outliers in durations */
+  outlierDurations?: number[];
```

Comment:
pls modify also `reassure-compare` package to add this field to validation (optional, as old files might not contain it). and to modify the output report to show removed outliers in the details table